### PR TITLE
tickets/SP-1250

### PR DESCRIPTION
**Pull request by MelissaGraham**

### CODE REVIEW

**Review by jeffcarlin — APPROVED** [review]
I think this is good to merge. I didn't explicitly test on the RSP (which was being _really_ slow when I tried to test it there), but I read through the changes and it looks good.

File: _static/nb_html/DP02_09a_Custom_Coadd.html [hunk 1]
```diff
@@ -14678,6 +14678,14 @@
 This is pedagogically undesirable because it does not accurately represent <em>future workflows</em>, which is the goal of DP0.2.
 Thus, it is recommended that delegates learn how to recreate coadds with Weekly 2022_40.</p>
 <hr />
+<p><strong>NOTICE:
+This notebook will only run with the butler repo alias <code>dp02-direct</code>.</strong></p>
+<p>All other tutorial notebooks in this repository use the <code>dp02</code> alias, which was updated in Oct 2024 to provide
+ready-only access to butler data repos.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply _static/nb_html/DP02_09a_Custom_Coadd.html:14684]: Fixed in both the 09a and 09b html files, thank you!

File: tutorials-examples/major-updates-log.rst [hunk 2]
```diff
@@ -51,6 +51,17 @@ and clicking on "history" (near upper-right).
 Major Updates Log
 =================
 
+Oct 24 2024
+-----------
+
+The ``dp02`` alias for the butler repository holding the DP0.2 data set was updated to be read-only.
+This change accommodates evolution in the butler service and better align with the functionality to be expected
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/major-updates-log.rst]: fixed

**Comment by MelissaGraham** [comment]
Ugh, forgot that resolving a conflict in order to merge counts as merging main into the branch, which just blocks the merge for a different reason. Will have to go back and uncommit and fix this...

**Pull request closed without merging**